doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,327 @@
+# Environment Variables
+
+Julia may be configured with a number of environment variables, either in the
+usual way of the operating system, or in a portable way from within Julia.
+Suppose you want to set the environment variable [`JULIA_EDITOR`](@ref) to
+`vim`, then either type `ENV["JULIA_EDITOR"] = "vim"` for instance in the REPL
+to make this change on a case by case basis, or add the same to the user
+configuration file `.juliarc.jl` in the user's home directory to have
+a permanent effect.  The current value of the same environment variable is
+determined by evaluating `ENV["JULIA_EDITOR"]`.
+
+The environment variables that Julia uses generally start with `JULIA`. If
+[`Base.versioninfo`](@ref) is called with `verbose` equal to `true`, then the
+output will list all environment variables for which `JULIA` appears in the
+name.
+
+## File locations
+
+### `JULIA_HOME`
+
+The absolute path of the directory containing the Julia executable, which sets
+the global variable [`Base.JULIA_HOME`](@ref). If `$JULIA_HOME` is not set, then
+`Base.JULIA_HOME` defaults to its compile-time value.
```

Comment:
(and sorry for not taking your comments into account, they didn't show up on the last version...)